guides/operator/orca-redis-to-sql.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: single
+title:  "Orca: Redis to SQL"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Orca's execution state is stored in Redis by default, but can be configured for SQL. In this topology, Redis will still be required for the work queue.
+
+This guide will go over MySQL setup, how to configure Orca, as well as how to perform a zero-downtime migration from Redis to SQL.
+
+## MySQL Setup
+
+Orca ships with MySQL drivers by default, but you can include your own JDBC drivers on the classpath if you need to connect to a different database.
+
+First, you'll need to create the schema:
+
+```sql
+CREATE SCHEMA `orca` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ;
+```
+
+Orca is configured to support a `service` user and a `migration` user:
+
+```sql
+GRANT SELECT, INSERT, UPDATE, DELETE, EXECUTE, SHOW VIEW ON `orca`.* TO 'orca_service'@'%'
+
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, INDEX, ALTER, LOCK TABLES, EXECUTE, SHOW VIEW ON `orca`.* TO 'orca_migrate'@'%'
+```
+
+When Orca starts up, it will perform database migrations to ensure its running the correct schema.
+
+### Amazon Aurora
+
+At Netflix, we use Amazon Aurora MySQL and have configured it with the following settings, missing values are defaults.
```

Comment:
Include major versions - i.e. Aurora MySQL 5.7 and perhaps note that we locally develop against or have also tested with OSS MySQL 5.7.